### DISCUSSION TITLE
How to put zwave controller into inclusion mode?

### DISCUSSION THREAD

**Discussion by batni** [zwave-js/node-red-contrib-zwave-js#291] [Q&A]
1. I have added the controller and configured it properly
2. I have added one node

Question: While the node red server is running, without node red UI, can I put node red controller to inclusion mode?

**Comment by marcus-j-davies** [comment]
Hi @batni 

The controller can only be put into inclusion via the UI.
https://github.com/zwave-js/node-red-contrib-zwave-js/wiki/User-Interface
